includes/woop.php

```diff
@@ -32,5 +32,7 @@ function wc_calypso_bridge_woop_init() {
 	if ( is_admin() ) {
 		require_once dirname( __FILE__ ) . '/woop/hide-onboarding.php';
 	}
+
+	require_once dirname( __FILE__ ) . '/woop/limit-payment-gateways.php';
```

Comment:
Bit of a ramble touching on this code made me think about: 

I wouldn't mind figuring out how to properly separate admin code, a `woop/admin` dir maybe? include all and rely on the include to do the is_admin check? probably some code wants to assume that without defining things the front end can access.